tests/js/setup-before-after.js

```diff
@@ -1,3 +1,8 @@
+import fetchMockJest from 'fetch-mock-jest';
+
+// Set Jest global fetchMock so we don't have to import fetchMock in every test.
+fetchMock = fetchMockJest;
```

Comment:
Let's put the global "instantiation" in `tests/js/setup-globals.js` as `global.fetchMock = fetchMockJest` (unless there's a reason we need to do it this way?).

I'm not entirely sure if the import would still be needed here once it's global, so give that a test and see.

Reply:
@aaemnnosttv I think the import of `fetch-mock-jest` has to happen during Jest's `setupFilesAfterEnv`, (or later), since `fetch-mock-jest` relies on `expect()` from Jest. But I can move the instantiation of `global.fetchMock = undefined` into `setup-globals.js` which would remove the need for the globals declaration in `jest.config.js` and the extra eslint.rc global config.

Reply:
```suggestion
// Set Jest global fetchMock so we don't have to import fetchMock in every test.
global.fetchMock = fetchMockJest;
```

Please also elaborate on the comment here to explain why it's set here and not in `tests/js/setup-globals.js`.